libraries/abstractions/common_io/test/test_scripts/adc/test_iot_adc_test.py

```diff
@@ -49,8 +49,9 @@ def __init__(self, serial, ip, login, pwd, csv_handler):
         self._pwd = pwd
         self._cr = csv_handler
 
-        print("Please input platform ADC reference voltage:")
-        self.platform_ref_voltage = float(input())
+        # print("Please input platform ADC reference voltage:")
+        # self.platform_ref_voltage = float(input())
```

Comment:
Just remove? Any reason to keep them with comment?

Reply:
It is only a quick fix. DUT reference voltage may change across platforms (1.8 or 3.3). We need to figure out a way to get it.